ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -31,82 +31,6 @@ resources:
       memory: 4000Mi
 test_binary_build_commands: .openshift-ci/dispatch.sh test-binary-build-commands
 tests:
-- as: style-checks
-  commands: .openshift-ci/dispatch.sh style-checks
-  container:
-    clone: true
-    from: root
-  secrets:
-  - mount_path: /tmp/vault/stackrox-stackrox-base
-    name: stackrox-stackrox-base
-- as: policy-checks
-  steps:
-    test:
-    - ref: stackrox-initial
-- as: mitre-bundles-checks
-  steps:
-    test:
-    - ref: stackrox-initial
-- as: grouped-static-checks
-  steps:
-    test:
-    - as: check-generated
-      commands: .openshift-ci/dispatch.sh check-generated
-      credentials:
-      - mount_path: /tmp/vault/stackrox-stackrox-initial
-        name: stackrox-stackrox-initial
-        namespace: test-credentials
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-    - as: check-todos
-      commands: .openshift-ci/dispatch.sh check-todos
-      credentials:
-      - mount_path: /tmp/vault/stackrox-stackrox-initial
-        name: stackrox-stackrox-initial
-        namespace: test-credentials
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-    - as: check-pr-fixes
-      commands: .openshift-ci/dispatch.sh check-pr-fixes
-      credentials:
-      - mount_path: /tmp/vault/stackrox-stackrox-initial
-        name: stackrox-stackrox-initial
-        namespace: test-credentials
-      from: src
-      resources:
-        requests:
-          cpu: 2000m
-          memory: 4000Mi
-- as: go-unit-tests
-  commands: .openshift-ci/dispatch.sh go-unit-tests
-  container:
-    clone: true
-    from: root
-  secrets:
-  - mount_path: /tmp/vault/stackrox-stackrox-base
-    name: stackrox-stackrox-base
-- as: go-unit-tests-release
-  commands: .openshift-ci/dispatch.sh go-unit-tests-release
-  container:
-    clone: true
-    from: root
-  secrets:
-  - mount_path: /tmp/vault/stackrox-stackrox-base
-    name: stackrox-stackrox-base
-- as: go-postgres-tests
-  commands: .openshift-ci/dispatch.sh go-postgres-tests
-  container:
-    clone: true
-    from: root
-  secrets:
-  - mount_path: /tmp/vault/stackrox-stackrox-base
-    name: stackrox-stackrox-base
 - as: integration-unit-tests
```

Comment:
[not a blocker]

@gavin-stackrox @janisz I see `integration-unit-tests` in GHA

https://github.com/stackrox/stackrox/blob/f9ef9a7a26b0d63c3bb4c8ecd9eacc8db5cf6ab7/.github/workflows/unit-tests.yaml#L39-L40

Do you want to disable them here or is the plan to fully migrate them later?

(the same comment applies to `merge-integration-unit-tests`)

Reply:
I think we should remove it

Reply:
I did not see them in the go-unit tests. Will remove.